ros2bag/ros2bag/verb/play.py

```diff
@@ -208,15 +208,17 @@ def main(self, *, args):  # noqa: D102
 
         play_options.exclude_regex_to_filter = args.exclude_regex
 
-        play_options.exclude_topics_to_filter = args.exclude_topics if args.exclude_topics else []
-
         play_options.exclude_service_events_to_filter = \
             convert_service_to_service_event_topic(args.exclude_services)
 
         play_options.topic_qos_profile_overrides = qos_profile_overrides
         play_options.loop = args.loop
         play_options.topic_remapping_options = topic_remapping
         play_options.clock_publish_frequency = args.clock
+        exclude_topics = args.exclude_topics if args.exclude_topics else []
+        if play_options.clock_publish_frequency > 0:
+            exclude_topics.append('/clock')
```

Comment:
i think we should explain this behavior in `--clock` option help field for sure, e.g `If specified, /clock topic in the bag file is excluded to publish.`.

Reply:
thanks!! I updated option help in https://github.com/ros2/rosbag2/pull/1646/commits/2961ccc8f377877bc4bfa081a9abb4a782fb237e cc @MichaelOrlov